containers/src/Kokkos_Vector.hpp

```diff
@@ -64,10 +64,10 @@ class vector : public DualView<Scalar*, LayoutLeft, Arg1Type> {
   typedef const Scalar& const_reference;
   typedef Scalar* iterator;
   typedef const Scalar* const_iterator;
+  typedef size_t size_type;
```

Comment:
```
using size_type = size_t;
```

Reply:
all the other ones are typedef right now. Somebody at some point can replace all typedefs with using if that someone feels its worth it ;-).